Assets/Scripts/Commands/MultimirrorDuplicateCommand.cs

```diff
@@ -81,13 +81,22 @@ public MultimirrorDuplicateCommand(TrTransform xf, bool stampMode, BaseCommand p
             m_DuplicatedWidgets = new List<GrabWidget>();
             foreach (var widget in m_SelectedWidgets)
             {
+                if (widget is Media2dWidget)
+                {
+                    ((Media2dWidget)widget).TwoSided = true;
+                }
                 TrTransform widgetTransform_GS = TrTransform.FromTransform(widget.transform);
                 TrTransform tr_GS;
                 var xfCenter_GS = TrTransform.FromTransform(PointerManager.m_Instance.SymmetryWidget);
+                // Generally speaking we want both sides of 2d media to appear
```

Comment:
Maybe I'm missing something obvious, but what's the difference between this boolean and the `.TwoSided` set a few lines earlier?

Reply:
Good catch. That's left over from a small refactor.